src/main/java/com/example/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,11 +1,18 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
         String title,
+        @NotNull
         String author,
+        @NotNull
```

Comment:
Also, you can validate isbn field by using pattern

Reply:
agree